src/Redux/userSlice.js

```diff
@@ -1,7 +1,7 @@
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 import axios from 'axios';
 
-const initialState = { entities: [], loading: 'idle' };
+const initialState = { entities: {}, loading: 'idle' };
```

Comment:
why is this called `entries` and not just `user` since it only has one key inside called `currentUser`

Reply:
This is a hold-over from when I was first getting used to using redux and very reluctant to deviate in any way from the boilerplate.  You're absolutely right though, I will remove this unneeded 'entities' nesting.